### PR TITLE
chore: Fix linter findings for prealloc (part2)

### PR DESCRIPTION
**Pull request by zak-pawel**
Address findings for [prealloc](https://github.com/alexkohler/prealloc) - Finds slice declarations that could potentially be pre-allocated.

It is only part of the bigger job.
After not pre-allocated slices in whole project are handled, we can enable proper linter to guard this.

### CODE REVIEW

File: plugins/inputs/nfsclient/nfsclient.go [hunk 1]
```diff
@@ -33,20 +33,19 @@ type NFSClient struct {
...
 	   the /proc/self/mountstats file, and returning faulty data
 	   is worse than no data.  Other errors are ignored, and append
 	   whatever we got in the first place (probably 0).
 	   Yes, this is ugly. */
 
-	var nline []uint64
-
 	if len(line) < 2 {
-		return nline, nil
+		return nil, nil
```
Comment (powersj) [inline plugins/inputs/nfsclient/nfsclient.go:45]: hmm where this is called will throw an error as it immediately calls `len()` on the return value and `len(nil)` won't work, right?

Comment (zak-pawel) [reply plugins/inputs/nfsclient/nfsclient.go:45]: `len` should return for `nil` slice

https://go.dev/play/p/U0v-Gk_5R11

Comment (powersj) [reply plugins/inputs/nfsclient/nfsclient.go:45]: ah right

**Review by powersj — APPROVED** [review]

**Review by srebhan — APPROVED** [review]
LGTM. Thanks for the effort @zak-pawel!

**Pull request merged**